pkg/daemon/ceph/osd/daemon.go

```diff
@@ -85,7 +102,7 @@ func Provision(context *clusterd.Context, agent *OsdAgent) error {
 	// determine the set of devices that can/should be used for OSDs.
 	devices, err := getAvailableDevices(context, agent.devices, agent.metadataDevice, agent.usingDeviceFilter)
 	if err != nil {
-		return fmt.Errorf("failed to get available devices. %+v", err)
+		return fmt.Errorf("failed to get available devices: %+v", err)
```

Comment:
did you find the error messages more readable with the colon? our pattern has been to put a sentence with each message that is appended.

Reply:
I have been a little confused when sometimes colons are used and sometimes periods. Most of what I've seen online uses a colon, so I think I assumed there was a best practice around that. I can change these back if you want to continue using periods.

Reply:
i'd summarize the pattern as:
- use a period when an err is appended with %+v
- use a colon when there is some data appended to the message

Reply:
Okay. I'll revert these then. I haven't been following that convention, so if I notice the colons in other PRs, I can replace them with periods

Reply:
I may have been a little heavy-handed with fixing this. It was hard to go through each instance I missed, and it was easier to do some semi-manual copy-replace following the rule that if an error follows a message there is a period, and if data follows a message there is a colon. I probably 'fixed' some things I didn't mess up in the first round.